### PR TITLE
Feature#22improves

### LINKED ISSUE #22: Rubocop improves
# REQUIREMENTS
Setting up rubocop to improve Ruby code

# DESIGN
Not apply

### PR DESCRIPTION
**Pull request by alexs**
# Description
What did you implemented/Why did you implemented this:
 - Setting up rubocop in the project
 - Refactor components based in rubocop suggestions
 - General improves
 - 
 Examples:
 - What?: Improve code quality based in ticket #22 
 - Why?: To improve quality

 # Testing
 rake db:test:prepare
 bundle exec rspec spec/models

  ## Screenshots
Not apply to this issue

### CODE REVIEW

File: app/models/attendance.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@
 #  updated_at  :datetime         not null
 #  employee_id :bigint           not null
 #
+# frozen_string_literal: true
+
```
Comment (dannharo) [inline app/models/attendance.rb:13]: lets try to add this magic comment at the beginning of the file.

File: config/puma.rb [hunk 2]
```diff
@@ -1,28 +1,30 @@
+# frozen_string_literal: true
```
Comment (dannharo) [inline config/puma.rb:1]: I think you could avoid edit these(config, etc...) rails files and only take a look of `"your files"`

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```
Comment (dannharo) [inline config/routes.rb:2]: 👍🏻

**Review by dannharo — COMMENTED** [review]
Very Good Job, just some comments

File: app/models/attendance.rb (hunk 1 above)
Comment (alexs) [reply app/models/attendance.rb:13]: Ok ok thanks!

File: config/puma.rb (hunk 2 above)
Comment (alexs) [reply config/puma.rb:1]: Ok ok we will put it in exceptions, I think we already made it, but we will review again.

File: config/routes.rb (hunk 3 above)
Comment (alexs) [reply config/routes.rb:2]: thx

**Comment by nallely64** [comment]
LGTM 🗡️

**Review by germanruzca — APPROVED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
LGTM

File: app/controllers/employees_controller.rb [hunk 4]
```diff
@@ -1,36 +1,37 @@
...
-    if @search
-      @employees = Employee.where("name LIKE ?", "%" + @search + "%")
-    else
-      @employees = Employee.order(:name)
-    end
+    @employees = if @search
+                   Employee.where('name LIKE ?', "%#{@search}%")
+                 else
+                   Employee.order(:name)
+                 end
```
Comment (AlejandroCuriel) [inline app/controllers/employees_controller.rb:14]: here, could you use the ternary operator?

**Review by nallely64 — APPROVED** [review]

**Review by AlejandroCuriel — CHANGES_REQUESTED** [review]
could be an opportunity to add the ternary operator

File: app/controllers/employees_controller.rb (hunk 4 above)
Comment (AlejandroCuriel) [reply app/controllers/employees_controller.rb:14]: please do not resolve this conversation if you answer first

**Pull request merged**